easybuild/easyblocks/generic/bundle.py

```diff
@@ -308,3 +335,12 @@ def sanity_check_step(self, *args, **kwargs):
             fake_mod_data = self.load_fake_module(purge=True)
             self.log.debug("Cleaning up after testing loading of module")
             self.clean_up_fake_module(fake_mod_data)
+
+        # run sanity checks for specific components
+        cnt = len(self.comp_cfgs_sanity_check)
+        for idx, comp in enumerate(self.comp_cfgs_sanity_check):
+            comp_name, comp_ver = comp.cfg['name'], comp.cfg['version']
+            print_msg("sanity checking bundle component %s v%s (%i/%i)...", comp_name, comp_ver, idx + 1, cnt)
```

Comment:
I wonder if the `(x/y)` is worthwile adding, since if not all components are being sanity checked, this may cause confusion why the total count mentioned here is lower than the total number of components?

Reply:
I felt it was nice to have an idea of the overall progress, but since the sanity checks shouldn't take overly long I'm happy to remove it, if it's likely to cause more confusion than it helps

Reply:
No strong objection, let's proceed as is, it's easy enough to change later if it's indeed confusing people.